src/scss/patterns/_media.scss

```diff
@@ -102,3 +102,12 @@
   @include tools.position(0);
   @include tools.square(100%);
 }
+
+[data-media='desktop-work'] {
```

Comment:
I can't find where this is being used?

Reply:
I thought it was on examples where we aren't showing a mobile screen, like on Workflow Builder Case Study in the what we do section?

Reply:
![Screen Shot 2021-09-13 at 11 27 58 AM](https://user-images.githubusercontent.com/1581694/133121710-29b89245-d3f9-401c-8b8d-d9f5813f43a4.jpg)

Reply:
@mirisuzanne @stacyk yeah, it's being used on the expression-builder, shopify and workflow-builder pages that don't have a mobile-specific sample.

Reply:
aha, ok, looks great!